data-pipeline/src/mappings.py

```diff
@@ -0,0 +1,200 @@
+import datetime
+
+import pandas as pd
+
+
+def map_ofsted_rating(rating: str):
+    match rating.lower():
```

Comment:
Lowercase string won't match either of the two cases.

Reply:
Actually deliberate for the minute, because I couldn't figure out what the correct mapping is... so wanted the default. Why I actually did it this way. No IDEA 🥇 but it is stupid